pageserve/app.py

```diff
@@ -22,24 +22,67 @@
 
 app = Flask(__name__)
 
-@app.route('/')
+
+@app.route('/v1/')
```

Comment:
I'd recommend moving this change to a separate PR, since it looks like it should be unrelated to the events service.

Reply:
I'm having trouble figuring out how to move changes from past commits to different PRs, is there a good approach to this?

Reply:
Maybe you could make a new branch from this branch (which will carry over all of its commits) then revert all the commits on each branch that shouldn't be on their respective branches if that makes sense.

Reply:
Yes, that's one way to do it.

If you're careful about your commits, you can use `git cherrypick` to copy over the commits. But I'm guessing that you don't really care about the commit history, so you can just:

```shell
git branch pr1  # Makes a new branch with the same commits as your current branch
git checkout upstream/master -- events # Reset the events files to what's in upstream/master
git commit -a -m "Reset events"
git checkout master  # Back to this branch
git checkout upstream/master -- pageserve  # Reset the pageserve files to what's in upstream/master
git commit -a -m "Reset pageserve"
git push origin
```